common/crypto_utils/avalon_crypto_utils/worker_signing.py

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/python3
```

Comment:
Existing file shown as new. Please correct

Reply:
Its because file is moved and new methods are added.

Reply:
Please create multiple commits. This file should only reflect the modification like in worker_hash.py and worker_encryption.py

Reply:
Done